GraphClient_test.go

```diff
@@ -43,7 +51,8 @@ func TestEnvironmentVariablesPresent(t *testing.T) {
 		t.Fatal("Environment Variable for an existing user in the group named <MSGraphExistingUserPrincipalInGroup> is missing!")
 	}
```

Comment:
I moved the whole initialization to a `TestMain` method and check everything before the tests are executed this way you don't need to check anything before any test and error handling is a little bit easier.
Also I replaced `strconv.Atoi` with `strconv.ParseUInt()` to avoid negative values directly when parsing :man_shrugging: